packages/job-worker/src/blueprints/context/lib.ts

```diff
@@ -121,10 +121,20 @@ function convertPieceInstanceToBlueprintsInner(pieceInstance: PieceInstance): Co
 	return obj
 }
 
+/**
+ * Convert a PieceInstance into IBlueprintPieceInstance, for passing into the blueprints
+ * @param pieceInstance the PieceInstance to convert
+ * @returns a cloned complete and clean IBlueprintPieceInstance
+ */
 export function convertPieceInstanceToBlueprints(pieceInstance: PieceInstance): IBlueprintPieceInstance {
 	return convertPieceInstanceToBlueprintsInner(pieceInstance)
```

Comment:
I have seen this naming convention throughout Sofie where `Inner` is postfixed to method names.
I'm curious: What is the purpose/benefit of this?

Reply:
I typically use it when the method is the same as the non `Inner` version, but takes more localised input parameters.
In this case, its because I want the return type of `convertPieceInstanceToBlueprintsInner` to be `Complete<IBlueprintPieceInstance>`, as the method is also used by `convertResolvedPieceInstanceToBlueprints`, but I want callers outside of the file to be given `IBlueprintPieceInstance`.
`Inner` may not be the best description, but I dont know what would be better. `EnsuringComplete`? without understanding the custom return type that isnt any clearer